utils/include/CXMLParser.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "pugixml.hpp"
+#include "IParser.h"
+
+class CXMLParser : public IParser
+{
+public:
+	CXMLParser() = default;
+	CXMLParser(CXMLParser&) = delete;
+	CXMLParser(CXMLParser&&) = delete;
+
+	~CXMLParser() override = default;
+
+	bool Initialize(const std::string& path_to_configuration_file) override;
+
+	SServer& TryToGetServerConfigurationFromFile() override;
```

Comment:
Looks like these methods should return a const reference

Reply:
fixed